Classes/rental.rb

```diff
@@ -6,9 +6,19 @@ def initialize(date, person, book)
     @date = date
 
     @person = person
-    person.rentals << self
+    p person
+    if person.instance_of?('Teacher') || person.instance_of?('Student')
```

Comment:
- [ ] You should be sure that you didn't break any of the functionality of the app. 

When I try to create a rental, the app crashes at this line.
![Screenshot from 2021-10-19 23-23-59](https://user-images.githubusercontent.com/1642014/137985840-d0a3c6ca-8112-40b8-ab86-42f01b7007cf.png)